cfgov/jinja2/v1/_includes/organisms/snippet-list.html

```diff
@@ -71,11 +71,19 @@
     {% if filtered_snippets.exists() %}
         <div class="o-table__stack-on-small o-snippet-list_list" data-display-table="table">
             <header data-display-table="thead">
-                <div class="o-snippet-list_list-heading" data-display-table="cell">Description</div>
+                <div class="o-snippet-list_list-heading" data-display-table="cell">
+                    Description
+                 </div>
             {% if value.show_thumbnails %}
                 <div class="o-snippet-list_list-heading" data-display-table="cell"></div>
             {% endif %}
-                <div class="o-snippet-list_list-heading" data-display-table="cell">Actions</div>
+                <div class="o-snippet-list_list-heading"
+                     data-display-table="cell"
+                     {% if value.actions_column_width %}
+                     style="width: {{ value.actions_column_width }}%;"
```

Comment:
Instead of inline CSS can we use https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-utilities.less#L229-L242 and provide a drop-down menu in the admin? Or do the end users need % more precise than that?

Reply:
Oh I like this idea. I'll give it a try.